tests/testthat/helper-DBItest.R

```diff
@@ -8,7 +8,7 @@ DBItest::make_context(
   ),
   tweaks = suppressWarnings(
     DBItest::tweaks(
-      dbitest_version = "1.7.3",
+      dbitest_version = "1.8.0",
```

Comment:
Where do I get this DBItest version from? I'm currently using main from r-dbi/DBItest which is at v1.7.3.9014 and this works ok. I don't understand the meaning of `dbitest_version`, as it does not complain about me only having v1.7.3.9014 available. Is this some versioning scheme only for tweaks such that if I specify a tweak that does not exist in the current DBItest version that is running, this does not cause issues? Then again, this is somewhat covered by having `...` behave as

> Unknown tweaks are accepted, with a warning

Reply:
Thanks. This meansthe mainline DBItest and all future changes that make it into the upcoming 1.8.0 version. Search in DBItest for `dbitest_version` to see other possible values.

It's for gradually introducing new tests without spreading them to all packages that use DBItest. Packages need to opt in to using new checks.

Happy to review a PR that improves DBItest documentation, or to discuss what such an improvement would need to look like.